### PR TITLE
[DOC] Server documentation

### PR DESCRIPTION
**Pull request by 31Nathan**
Improving the documentation of the server

### CODE REVIEW

File: Server/controllers/Web/messages.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
 const { db } = require('../../middleware/database')
 
+/**
+ * Delete a message by its id
+ *
+ * @param {number} id of the message to be deleted
```
Comment (tbellicha) [inline Server/controllers/Web/messages.js:6]: No throws ?

**Review by tbellicha — CHANGES_REQUESTED** [review]

File: Server/controllers/Common/container.js [hunk 2]
```diff
@@ -61,13 +81,12 @@ exports.getAllContainers = async () => {
...
-  return db.Containers.create({
-    data: container,
-  });
-};
-
+/**
+ * Retrieve the items of the containers
+ *
+ * @param {number} containerId of the container object
+ * @returns the container object with its items
```
[the remark this answers is not in the document]
Comment (31Nathan) [reply Server/controllers/Common/container.js:88]: Yes it return the container with items on .items attributes

File: Server/controllers/Mobile/user.js [hunk 3]
```diff
@@ -85,6 +97,12 @@ exports.setNewUserPassword = (user, newPassword) =>{
   })
 }
 
+/**
+ * Delete an user from the database
+ *
+ * @param {number} id of the user to be deleted
+ * @returns none
```
Comment (CedricCORGE) [inline Server/controllers/Mobile/user.js:104]: if none is return why is it a return in the method ?

File: Server/documentation.example.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/**
+ * Example function for documentation
+ *
+ * @param {number} nbr to be returned @default 42
+ * @returns the nbr given in param
```
Comment (CedricCORGE) [inline Server/documentation.example.js:5]: in sentence it's better to say number than the variable name "nbr"

**Review by CedricCORGE — APPROVED** [review]

**Review by H-chauvet — APPROVED** [review]
Good for me

**Review by tbellicha — APPROVED** [review]

**Pull request merged**